DEPS

```diff
@@ -236,7 +236,7 @@ allowed_hosts = [
 ]
 
 deps = {
-  'src': 'https://github.com/flutter/buildroot.git' + '@' + 'b2ab6e1908b3eb268d2a2cab1ec5b63f38e1bc11',
+  'src': 'https://github.com/flutter/buildroot.git' + '@' + 'fbaac6aad455f77ccafc4d3f15bad31856fbcb5b',
```

Comment:
Double check that this won't revert someone else's buildroot roll =)

Reply:
Thanks for the reminder! Yes, this needs to change to whatever SHA is produced once the [buildroot PR](https://github.com/flutter/buildroot/pull/663) lands.